subprojects/docs/src/docs/userguide/distributionPlugin.adoc

```diff
@@ -110,44 +110,68 @@ For each extra distribution set you add to the project, the distribution plugin
 
 ++++
 <sample id="multipleDistribution" dir="userguide/distribution" title="Adding extra distributions">
-            <sourcefile file="build.gradle" snippet="custom-distribution"/>
-        </sample>
+    <sourcefile file="build.gradle" snippet="custom-distribution"/>
+</sample>
 ++++
 
-This will add following tasks to the project: 
+This will add following tasks to the project:
```

Comment:
❓ would this be better formatted inline, without a bulleted list?

Reply:
✅ Done.